### DISCUSSION TITLE
Hobby plan not available in India

### DISCUSSION THREAD

**Discussion by therohitdas** [planetscale/discussion#602] [FAQs]
Hi, can someone let me know when the Hobby Plan will return for Indian users?

I was creating a new account for my company, and I have a company VISA debit card.
I have not added the debit card to the billing. Does adding the card enable the Hobby Plan for Indians?

Currently, it says:
> Hobby databases are currently not available to users in your country.

Also, does this change affect databases already created by Indian users?

**Comment by orware** [comment]
Hello @therohitdas,

This change should not affect already created databases at this time.

As far as a timeline that isn't something that I have available right now, but considering that this is a very recent change I wouldn't expect any adjustments in the near future since we are currently optimizing for markets that we are set up to serve well as a company. 

In the future we will be bringing the free Hobby plan back in your region once we are set up to deliver our full range of services within it.

The new restriction currently only applies to Hobby plan databases in your location however, and users are still welcome to create paid plan databases should they need to.

Our existing requirement to add a bank issued debit or credit card, with prepaid cards not being allowed, also still applies for all user accounts.

I hope this helps to answer your questions!

**Reply by therohitdas** [reply]
Thank you very much for the detailed answer.

Are you guys open to creating a paid hobby plan for now? 29$ plan is too steep for countries like India.
This will help small projects in regions without a free hobby plan.

Something like 9$ is a good price point for it.
